src/Compiler.php

```diff
@@ -218,6 +227,6 @@ public function outdent(int $step = 1)
 
     public function getVarName(): string
     {
-        return sprintf('__internal_compile_%d', $this->varNameSalt++);
+        return \sprintf('__internal_compile_%d', $this->varNameSalt++);
```

Comment:
The root namespace is added only for functions that have a special opcode compilation. This is done by php-cs-fixer. 
https://cs.symfony.com/doc/rules/function_notation/native_function_invocation.html

The function sprintf don't need to be root-namespaced.

Reply:
Rule native_function_invocation¶
Adds a leading \ before the function call to speed up resolution.

Warning¶
Using this rule is risky¶
Risky if one of the functions is overridden.

Well, then you should never use rule native_function_invocation, as this could always be risky depending on the context, right?

Reply:
The point is not about the rule being risky but about the fact that the rule as applied here only prefixes some functions that can be optimized by the compiler with a backslash.

Reply:
This is not objected to by the php cs fixer. This link https://stackoverflow.com/questions/55419673/php7-adding-a-slash-to-all-standard-php-functions-php-cs-fixer-rule shows that it is generally not that simple.